packages/core/android/src/newarch/src/DdLogs.kt

```diff
@@ -0,0 +1,148 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.ReadableMap
+
+/**
+ * The entry point to use Datadog's Logs feature.
+ */
+class DdLogs(
```

Comment:
suggestion: maybe it is worth to name this class `DdLogsSpec` or `DdLogsModule` and keep the old class `DdLogs` without renaming to `DdLogsImplementation`? wdyt? not important point for me though, just for the consistency with the base class naming.

Reply:
That's a fair point. 
I'd see one reason to keep it as it is now, it's that it respects the naming in all the new architecture examples. That would help anyone looking at this to understand which file is doing what more easily (and we can also maybe have the SDK showcased in migrations ;)).
So I'd be in favour of keeping the naming as it is for now, and maybe switch later when the new architecture is more widely spread, what do you think?

Reply:
I'm fine with that, I just didn't enjoy having `Implementation` suffix, but I guess we can live with that.